### PR TITLE
Extension: Library.on_init() must be invoked earlier

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
Addresses #825.

Make sure library.on_init() is called only once

### CODE REVIEW

File: src/taipy/gui/gui.pyi [hunk 1]
```diff
@@ -219,6 +220,7 @@ class Gui:
     def _hold_actions(self, callback: t.Optional[t.Union[str, t.Callable]] = ..., message: t.Optional[str] = ...): ...
     def _resume_actions(self) -> None: ...
     def _navigate(self, to: t.Optional[str] = ..., tab: t.Optional[str] = ..., force: t.Optional[bool] = ...): ...
+    def __init_libs(self) -> None: ...
```
Comment (FabienLelaquais) [inline src/taipy/gui/gui.pyi:222]: Do we really want the `_` functions and methods in the pyi file?

**Review by FabienLelaquais — APPROVED** [review]
Tricky one this one. Well done!

File: src/taipy/gui/gui.pyi (hunk 1 above)
Comment (FredLL-Avaiga) [reply src/taipy/gui/gui.pyi:222]: I don't think so but @dinhlongviolin1 knows better ;-)

Comment (dinhlongviolin1) [reply src/taipy/gui/gui.pyi:222]: Lets keep it for now, will come up with new process later (probably this week)

**Review by dinhlongviolin1 — APPROVED** [review]

**Pull request merged**